diff --git a/node_modules/pdf-parse/index.js b/node_modules/pdf-parse/index.js
index e9fc367..0a3066b 100644
--- a/node_modules/pdf-parse/index.js
+++ b/node_modules/pdf-parse/index.js
@@ -3,13 +3,13 @@ const Pdf = require('./lib/pdf-parse.js');
 
 module.exports = Pdf;
 
-let isDebugMode = !module.parent; 
+// let isDebugMode = !module.parent; 
 
 //process.env.AUTO_KENT_DEBUG
 
 
 //for testing purpose
-if (isDebugMode) {
+/* if (isDebugMode) {
 
     let PDF_FILE = './test/data/05-versions-space.pdf';
     let dataBuffer = Fs.readFileSync(PDF_FILE);
@@ -24,3 +24,4 @@ if (isDebugMode) {
     });
 
 }
+ */
\ No newline at end of file
